tensorflow_addons/activations/rrelu.py

```diff
@@ -44,19 +44,25 @@ def rrelu(
         training: `bool`, indicating whether the `call`
         is meant for training or inference.
         seed: `int`, this sets the operation-level seed.
+        gs: A `Generator`. Default value is tf.random.get_global_generator().
     Returns:
         result: A `Tensor`. Has the same type as `x`.
     """
     x = tf.convert_to_tensor(x)
+    lower = tf.cast(lower, x.dtype)
+    upper = tf.cast(upper, x.dtype)
+    if gs is None:
+        gs = tf.random.get_global_generator()
```

Comment:
Here we affect the global context, unlike before were the random global context wasn't affected. This is a breaking change potentially, especially if it's used with `reset_from_seed`. Also from the [documentation](https://www.tensorflow.org/api_docs/python/tf/random/get_global_generator) :

> This function will create the global generator the first time it is called, and the generator will be placed at the default device at that time, so one needs to be careful when this function is first called. Using a generator placed on a less-ideal device will incur performance regression.

So I believe that the behavior the rrelu should have when `gs` is None is the old one. We call tf.random.uniform and set the seed. 

If the user provides his/her generator, then we use that (gs.uniform(...))

If the user provides a seed and a generator, we throw a `ValueError` because it makes no sense, the user is likely mistaken, with a nice error message of course.

Reply:
> If the user provides a seed and a generator, we throw a `ValueError` because it makes no sense, the user is likely mistaken, with a nice error message of course.

I remove gs or seed. But it casues unstable result when I test the code.